packages/compass-indexes/src/components/search-indexes-modals/base-search-index-modal.tsx

```diff
@@ -260,25 +360,39 @@ export const BaseSearchIndexModal: React.FunctionComponent<
                 <br />
                 {mode === 'create' && (
                   <Body>
-                    By default, search indexes will have the following search
-                    configurations. You can refine this later.
+                    By default,{' '}
+                    {searchIndexType === 'vectorSearch'
+                      ? 'vector search'
+                      : 'search'}{' '}
+                    indexes will have the following search configurations. You
+                    can refine this later.
                   </Body>
                 )}
                 <Link
-                  href="https://www.mongodb.com/docs/atlas/atlas-search/tutorial/"
+                  href={
+                    searchIndexType === 'vectorSearch'
+                      ? 'https://www.mongodb.com/docs/atlas/atlas-vector-search/vector-search-tutorial/'
+                      : 'https://www.mongodb.com/docs/atlas/atlas-search/tutorial/'
+                  }
```

Comment:
I'll double check with product folks on these links before merging.

Reply:
Asked on slack: https://mongodb.slack.com/archives/C05C8MVJJQ7/p1707156642543119 as these links are probably fine for now I think I'll merge this pr and then update in a follow up if there are changes.